packages/playwright/src/common/ipc.ts

```diff
@@ -125,6 +125,7 @@ export type DonePayload = {
   fatalErrors: TestInfoErrorImpl[];
   skipTestsDueToSetupFailure: string[];  // test ids
   fatalUnknownTestIds?: string[];
+  missingProjectById?: string;
```

Comment:
I'd call this `fatalUnknownProjectId` to be consistent, and make it a boolean instead.